### PR TITLE
🐛 SAT: Fix test_oneof_usage fail #9860

### LINKED ISSUE #9860: 🐛 SAT: Fix `test_oneof_usage` fail
From the slack conv:
https://airbytehq.slack.com/archives/C02URF2BKL1/p1643341400367279

*Tell us what happens.*
After release of #9768 `test_oneof_usage` is failing with error:
```python
       for variant_path in variant_paths:
>           top_level_obj = dpath.util.get(self._schema, "/".join(variant_path[:-1]))
E           AttributeError: 'TestSpec' object has no attribute '_schema'
```

## Expected Behavior
`test_oneof_usage` shouldn't crash the test execution

### PR DESCRIPTION
**Pull request by keu**
## What
#9768 introduced bug:
 `dpath.util.get(self._schema, path)` does not exist in the test scope because this line of code is from `JsonSchemaHelper` code.

## How
replace with `helper.get_node` call

## Recommended reading order
1. `x.java`
2. `y.python`

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -11,11 +11,17 @@ repos:
     rev: 21.11b1
     hooks:
       - id: black
+        args: ["--line-length=140"]
```
Comment (keu) [inline .pre-commit-config.yaml:14]: pre-commit config was broken recently

**Review by vitaliizazmic — APPROVED** [review]

**Review by htrueman — APPROVED** [review]

**Pull request merged**